haskell/test/Test/FA2/Common.hs

```diff
@@ -38,3 +42,9 @@ setSimplePosition cfmm liquidityProvider lowerTickIndex upperTickIndex = do
         , sppDeadline = deadline
         , sppMaximumTokensContributed = PerToken 1000000 1000000
         }
+
+checkView
+  :: forall viewVal parameter
+   . (IsoValue viewVal, Eq viewVal, Show viewVal)
+  => TAddress parameter -> Text -> MD.ViewParam -> viewVal -> IntegrationalScenario
+checkView = MD.checkView sMetadata
```

Comment:
Don't think we need this helper, `checkView` is only used in one place anyway.

Reply:
Make sense, updated this.